meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -608,7 +608,7 @@ DISTRO_FEATURES:append:tegra = " opengl"
 DISTRO_FEATURES_DEFAULT:remove:tegra = "modsign"
 PREFERRED_PROVIDER_virtual/kernel:tegra ?= "linux-tegra"
 IMAGE_TEGRAFLASH_FS_TYPE:sota:tegra = "ota-ext4"
-INITRAMFS_FSTYPES:remove:tegra = "cpio.gz.cboot cpio.gz.cboot.bup-payload"
+INITRAMFS_FSTYPES:remove:tegra = "cpio.gz.cboot.bup-payload"
```

Comment:
We can probably let as before here and create a tegra-initrd-flash-initramfs.bbappend adding what is missing.

Reply:
The issue with the `tegra-initrd-flash-initramfs` is because the `image_types_tegra.bbclass` expect a `*.cboot` initrd but as we remove here is not possible to add it again.
To add the `cpio.gz.cboot` only for the `tegra-initrd-flash-initramfs` we need a new variable here
```
INITRAMFS_FSTYPES_REMOVE = "cpio.gz.cboot cpio.gz.cboot.bup-payload"
INITRAMFS_FSTYPES:remove:tegra = "$INITRAMFS_FSTYPES_REMOVE"
```
and on the tegra-initrd-flash-initramfs.bbappend we override to don't drop the `*.cboot` initrd
```
INITRAMFS_FSTYPES_REMOVE = "cpio.gz.cboot.bup-payload"
```